examples/models/llama/source_transformation/quantize.py

```diff
@@ -70,6 +72,69 @@ def quantize(  # noqa C901
     if qmode == "int8":
         # Add quantization mode options here: group size, bit width, etc.
         return WeightOnlyInt8QuantHandler(model).quantized_model()
+    elif qmode.startswith("torchao:"):
+        import glob
+        import os
+
+        libs = glob.glob(
+            os.path.abspath(
+                os.path.join(
+                    os.path.dirname(__file__),
+                    "../../../../cmake-out/third-party/ao/torchao/experimental/libtorchao_ops_aten.*",
```

Comment:
uh this hardcoded path is not ideal. Can we add install() in torchao/experimental/CMakeLists.txt and then we can find the installed library in CMAKE_INSTALL_PREFIX?

Reply:
If I understand, you want the path to be something like:

> "{CMAKE_INSTALL_PREFIX}/lib/libtorchao_ops_aten.*"

How is the python script going to know what CMAKE_INSTALL_PREFIX is?  Do you want to user to define this as an environment variable?